ext/js/display/display.js

```diff
@@ -1112,13 +1114,60 @@ class Display extends EventDispatcher {
                 }
                 button.disabled = !canAdd;
                 button.hidden = false;
+
+                if (displayTags !== 'never' && Array.isArray(noteInfos)) {
+                    this._setupTagsIndicator(i, noteInfos);
+                }
             }
             if (noteId !== null) {
                 this._viewerButtonShow(i, noteId);
             }
         }
     }
 
+    _setupTagsIndicator(i, noteInfos) {
+        const tagsIndicator = this._tagsIndicatorFind(i);
+        if (tagsIndicator === null) {
+            return;
+        }
+
+        const {tags: optionTags, displayTags} = this._options.anki;
+        const noteTags = new Set();
+        for (const {tags} of noteInfos) {
+            for (const tag of tags) {
+                noteTags.add(tag);
+            }
+        }
+        if (displayTags === 'non-standard') {
+            for (const tag of optionTags) {
+                noteTags.delete(tag);
+            }
+        }
+
+        if (noteTags.size > 0) {
+            tagsIndicator.disabled = false;
+            tagsIndicator.hidden = false;
+            tagsIndicator.title = `Card tags: ${Array.from(noteTags).join(', ')}`;
```

Comment:
`[...noteTags]` instead of `Array.from(noteTags)` (that's the more usual convention for yomichan)